api/card2k/exchange_card.py

```
@@ -0,0 +1,128 @@
+        self.partner_id = get_partner_id()
+        self.partner_key = get_partner_key()
+        self.provider = get_config_value("provider", "https://card2k.com")
+
+    def get_fee_exchange_card(self) -> dict:
+        """
+        API: Lấy phí đổi thẻ cào
+        """
+
+        url = f"{self.provider}/chargingws/v2/getfee?partner_id={self.partner_id}"
+        try:
+            response = requests.get(url)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)